model/common/src/icon4py/model/common/grid/grid_manager.py

```diff
@@ -398,12 +399,10 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         if exc_type is FileNotFoundError:
             raise FileNotFoundError(f"gridfile {self._file_name} not found, aborting")
 
-    def __call__(self, backend: Optional[gtx_backend.Backend], limited_area=True):
+    def __call__(self, backend: Optional[gtx_backend.Backend]):
```

Comment:
Unrelated to this PR, but it would be great if you could add  the return type hint...